lib/ansible/plugins/httpapi/ftd.py

```diff
@@ -120,13 +131,27 @@ def logout(self):
             'access_token': self.access_token,
             'token_to_revoke': self.refresh_token
         }
-        self.connection.send(
-            self._get_api_token_path(), json.dumps(auth_payload), method=HTTPMethod.POST,
-            headers=BASE_HEADERS
-        )
+
```

Comment:
As value returned by _get_api_token_path  is static and will not change over one playbook execution . Do you want to print it only during login method ?